civihr_default_theme/template.php

```diff
@@ -14,6 +14,169 @@
 require_once dirname(__FILE__) . '/includes/view.inc';
 
 /**
+ * Implements template_preprocess_views_view_table().
+ */
+function civihr_default_theme_preprocess_views_view_table(&$variables) {
+  $variables['classes_array'] = array_diff($variables['classes_array'], array('table-striped', 'table-bordered'));
+}
+
+/**
+ * Implements theme_table().
+ */
+function civihr_default_theme_table($variables) {
```

Comment:
this code looks *massive*, did you copy it from somewhere else? Please split it into smaller functions, then i'll re-review